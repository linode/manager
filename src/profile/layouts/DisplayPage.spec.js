describe('profile/DisplayPage', () => {
  it.skip('should renders without error', () => {
  });
});
