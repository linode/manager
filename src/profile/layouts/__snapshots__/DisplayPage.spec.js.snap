// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`profile/DisplayPage should renders without error 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DisplayPage
    dispatch={[Function]}
    profile={
        Object {
            "email": "user@address.com",
            "timezone": "US/Eastern",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <section>
          <ChangeTimezone
                    dispatch={[Function]}
                    timezone="US/Eastern"
          />
</section>,
        <ChangeEmail
          dispatch={[Function]}
          email="user@address.com"
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <ChangeTimezone
            dispatch={[Function]}
            timezone="US/Eastern"
/>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "dispatch": [Function],
            "timezone": "US/Eastern",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "section",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "dispatch": [Function],
          "email": "user@address.com",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <section>
            <ChangeTimezone
                        dispatch={[Function]}
                        timezone="US/Eastern"
            />
</section>,
          <ChangeEmail
            dispatch={[Function]}
            email="user@address.com"
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <ChangeTimezone
              dispatch={[Function]}
              timezone="US/Eastern"
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "dispatch": [Function],
              "timezone": "US/Eastern",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "dispatch": [Function],
            "email": "user@address.com",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
